db_setup.py

```diff
@@ -1,36 +1,47 @@
-import os
-import sys
-from sqlalchemy import Column, ForeignKey, Integer, String, DateTime, func, Float
-from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import relationship
-from sqlalchemy import create_engine
-
-Base = declarative_base()
-
-
-class Violation(Base):
-    __tablename__ = 'violation'
-
-    id = Column(Integer, primary_key=True)
-    car_id = Column(Integer)
-    longitude = Column(Float)
-    latitude = Column(Float)
-    speed = Column(Float)
-    time = Column(DateTime)
+from pymongo import MongoClient
+
+def _create_database(URI,db_name):
+    """
+        Warning: This function is intended only to be used inside the scope of this file
+        you should not use it outside
+        Usage: _create_database(URI,db_name) return a database object
+        Example: db = _create_database(URI,db_name)
+    """
+    clinet = MongoClient(URI)
+    return clinet[db_name]
+
+class data_manager():
+    """
+        This class was a made as wrapper for PyMongo libraries to add some constrain on collection names and to hide mango layer to be replaceable 
+        Usage:
+        Example:
+    """
+    def __init__(self,URI="mongodb://localhost:27017/",
+                 db_name="evil_car"):
+        self.db = _create_database(URI,db_name)
+
+    def _get_collection(self,collection_name):
+        return self.db[collection_name]
+
+    def add_data(self,collection_name,data):
+        """
+        """
+        # TODO: add data checkers here
```

Comment:
for the checkers look at my comment at name.py
if u used dictionary you check its fields easly

Reply:
i forgot to remove this TODO though, checkers are not appropriate here because add data should do one thing **adding data**.